auto_submit/test/service/validation_service_test.dart

```diff
@@ -691,7 +691,33 @@ void main() {
         messagePullRequest: pullRequest,
       );
 
-      expect(result.message, 'Reland "My first PR!"');
+      expect(result.message, contains('Reland "My first PR!"'));
+    });
+
+    test('Includes PR description', () async {
+      final PullRequest pullRequest = generatePullRequest(
+        prNumber: 0,
+        repoName: slug.name,
+        title: 'PR title',
+        // This will be interpolated into JSON, so escape newlines
```

Comment:
can you elaborate on this comment?

Reply:
this is specifically for the test helper function `generatePullRequest()`.

Reply:
I can flesh out the comment

Reply:
done